package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "covid-assessment-centres",
+	"version": "0.1.0",
+	"description": "Contain data files for Ontario COVID-19 assessment centre locations",
+	"scripts": {
+		"lint": "grunt lint"
+	},
+	"author": "Inclusive Design Research Centre",
+	"license": "BSD-3-Clause",
+	"dependencies": {
+		"fluid-grunt-lint-all": "^2.0.0"
+	},
+	"devDependencies": {
+		"grunt": "^1.3.0"
```

Comment:
Do you still pin dependencies? If so you'll want to remove the `^` from the front of the version number.
Also you should probably pull in eslint-config-fluid as a dependency. Sometimes these things don't make their way through properly when they are sourced through sub dependencies.